### ISSUE TITLE
Marketing content on Amazon download pages isn't localized

### ISSUE THREAD

**Issue by liuche** [mozilla-mobile/firefox-tv#728]
It seems like the download pages for Firefox TV aren't localized for all locales. See [email thread](https://groups.google.com/a/mozilla.com/d/msg/firefox-tv/95c2qfoPVik/uLjpQANnBgAJ) from @Delphine :

While it seems parts of our Firefox for Fire TV pages appear localized, definitely not all of it is. More specifically, what remains in English are these sections:
* The title where our product name appears at the top of the page
* Latest update (what's new) 
* Product features
* Product description

**Comment by Sdaswani** [comment]
@sdaswani to follow up with Amazon.

**Comment by Sdaswani** [comment]
@Delphine @liuche 
![screen shot 2018-03-27 at 2 21 00 pm](https://user-images.githubusercontent.com/4879954/37995666-1b0729ba-31ca-11e8-9df0-eb47e23e69e1.png)
![screen shot 2018-03-27 at 2 20 01 pm](https://user-images.githubusercontent.com/4879954/37995667-1b1f8942-31ca-11e8-9cf6-843de645a7b3.png)

I think we can localize everything we need to, like we have for german:

**Comment by Delphine** [comment]
@Sdaswani : As for the French website, I think the translation was done and uploaded manually by someone in staff through the dev website - but that's just me assuming. Would be great though to confirm that in case I've missed something and Amazon has some kind of machine translation in place already for all these sections.

Continuing from the email thread: Chenxia asks if we should add this to the l10n marketing repo, I assume we're talking about appstores. This means hooking up a new project to the web application managing it (stores_l10n) - and I'd then manage pushing the English strings out and afterwards exposing them for localization. That's what I currently do for "What's New" - as well as all the other stores content updates like listing, title, keywords, etc. - on Firefox Android and Firefox iOS.

Since the releases seem to be frequent for Firefox TV, it means a lot of overhead for us if we had to manage all the sections called out in the 1st comment (stores_l10n updates take a lot of time) and our team's engineers don't have the bandwidth to prioritize automating this process while they're migrating Gecko to the Fluent l10n framework (which is one of the reason we stopped localizing What's New content for all other mobile projects, other than for both Fennecs). However, one option is that I could do like we currently do for Focus Android/iOS, which is a one-time set up of all the sections except "What's New" and "Product features" + tags/keywords if needed (since those are the ones that will be updated regularly) - and possibly have SUMO maintain the "What's New" and "Product Description" since this is marketing content. I'm also open to other suggestions.

On a side note, we're still reaching out to other localization contacts in Amazon in case they have any solutions on their side. Please let me know Susheel in case you get a hold of them before me :D
thanks everyone!

**Comment by Sdaswani** [comment]
@delphine yes the French listing was changed by me at the request of Sylvestre. Sorry I didn’t change the app name - I think that caused the initial confusion!😫

I’m fine with your option to set up Firefox TV similar to Focus Android / iOS 👍🏾.

Please do let me know if your Amazon contacts bear more fruit than I have so far, but I’ll keep pushing in my side. 🙏🏾

**Comment by Sdaswani** [comment]
@Delphine it looks like there is an [API for app publishing](https://developer.amazon.com/developer-publishing), and you can [specify localized conten](https://developer.amazon.com/docs/dev-publishing-api/devpubapi-update-an-existing-app.html#sample-json-to-put)t. Can you check this out and let us know if you have questions.

**Comment by Delphine** [comment]
Thanks Susheel! Will definitely look into this asap.

**Comment by Delphine** [comment]
Sorry, took me time to get back about this.
So I took a look and talked around, and turns out that it would actually be Firefox TV engineers who would know if this works or not :)

This might help, it's what what we're currently doing for Appstore and Google Play:
https://l10n.mozilla-community.org/stores_l10n/documentation/
I know @st3fan has done the entire Appstore integration, so he may have some further docs or info.

To sum things up, we can provide the translations in either case but your team should see: 
a) if you can use the API 
b) if you just want us to hand in translated content so that you can then use it (so we'd need to know the format you need it in, etc)
c) how to handle the What's New/Product section since that's something we can't currently update on a regular basis (explained in my previous comment above)

I'm CCing @flodolo here so he stays in the loop, since depending on what route we go down, he may need to be somewhat involved (sorry flod ;) ). I know this is a lot of information at once, let me know if you'd rather just jump on a call to go over this.

**Comment by Sdaswani** [comment]
Thanks @Delphine - I think the immediate thing we need to do is have @bbinto prioritize this. Barbara let me know if you want to discuss IRL to help you prioritize.

**Comment by bbinto** [comment]
Sorry for the huge delay.

Unfortunately, I need to ask some questions before prioritizing this

> a) if you can use the API

@Sdaswani , could you please have the engineers do a high-level check/estimation if they can use this API with FFTV?

> b) if you just want us to hand in translated content so that you can then use it (so we'd need to know the format you need it in, etc)

That probably is not maintainable in the long run, so I would say no, or only for the top 5 locales.

> c) how to handle the What's New/Product section since that's something we can't currently update on a regular basis (explained in my previous comment above)

Sorry I need to clarify/verify: We currently don't have a "What's New Page" for FFTV (yet). What content do you refer to when you say "product section", @Delphine?

**Comment by bbinto** [comment]
Also to clarify, @Sdaswani we cannot submit the app if we don't have an English, a French and German product page, correct (as shown as a screenshot in your comment above)?

**Comment by Delphine** [comment]
@bbinto: I mean the What's New / Product sections that appear on the amazon website (attached screenshots)
![screen shot 2018-05-04 at 10 01 02 am](https://user-images.githubusercontent.com/399020/39641172-23f285e4-4f82-11e8-8338-5b6ff16ced6d.png)
![screen shot 2018-05-04 at 10 00 24 am](https://user-images.githubusercontent.com/399020/39641173-2414f1b0-4f82-11e8-99f8-029a1e9f9b23.png)

**Comment by Sdaswani** [comment]
@bbinto this will take at least 2-4 weeks for one engineer to build this automation tool (includes dev, testing, tooling, etc.)

We do need all of our marketing content localized to English French and German - yes.
